src/apis/vaults.ts

```diff
@@ -279,6 +286,53 @@ export class DefaultVaultsAPI {
         return theftReports.isEmpty;
     }
 
+    async getLiquidationCollateralThreshold(): Promise<u128> {
+        return await this.api.query.vaultRegistry.liquidationCollateralThreshold();
+    }
+
+    async getPremiumRedeemThreshold(): Promise<u128> {
+        return await this.api.query.vaultRegistry.premiumRedeemThreshold();
+    }
+
+    async getFees(_vaultId: AccountId): Promise<PolkaBTC> {
+        // TODO: get real value from backend
+        return this.api.createType("FixedU128", 368) as PolkaBTC;
+    }
+
+    async getAPY(vaultId: AccountId): Promise<string> {
+        const fees = await this.getFees(vaultId);
+        // TODO: convert from PolkaBTC to USD
+        const feesInUSD = fees;
+
+        const lockedDot = await this.collateralAPI.balanceLockedDOT(vaultId);
+        // TODO convert from DOT to USD
+        const lockedUsdValue = lockedDot;
+
+        return feesInUSD.div(lockedUsdValue).toString();
+    }
+
+    async getSLA(_vaultId: AccountId): Promise<number> {
+        // TODO: get real value from backend
+        return 62;
+    }
+
+    async getSlashableCollateral(vaultId: AccountId): Promise<string> {
```

Comment:
This logic will move to the parachain as well, so a simple getter here is sufficient. But we'll need the input PolkaBTC amount.

Reply:
Don't we already know the staked relayer's collateral? I'm not sure why we need a PolkaBTC input

Reply:
This is to calculate how much the vault can be slashed for a certain redeem request. Say Alice tries to redeem 1 PolkaBTC. It will depend on the vault's SLA how much collateral will be slashed for that specific request.